errbot/bootstrap.py

```diff
@@ -64,6 +64,8 @@ def bot_config_defaults(config):
         config.TEXT_DEMO_MODE = True
     if not hasattr(config, 'BOT_ADMINS'):
         raise ValueError('BOT_ADMINS missing from config.py.')
+    if not hasattr(config, 'BOT_ADMINS_NOTIFICATIONS'):
+        config.BOT_ADMINS_NOTIFICATIONS = ()
```

Comment:
One last very small thing: can we dafault it to BOT_ADMINS here ? So if someone doesn't touch anything he or she can still get those ?